searchlib/src/vespa/searchlib/engine/packetconverter.cpp

```diff
@@ -117,13 +117,15 @@ PacketConverter::fromSearchReply(const SearchReply &reply, QUERYRESULTX &packet)
         packet.AllocateGroupData(reply.groupResult.size());
         memcpy(packet._groupData, &(reply.groupResult[0]), reply.groupResult.size());
     }
-    packet._features |= QRF_COVERAGE;
+    packet._features |= QRF_COVERAGE | QRF_EXTENDED_COVERAGE;
```

Comment:
this should be in the packet itself, since this is where the optional behavior has become mandatory

Reply:
Fixed